docs/kube-api-custom-ocp-versions.md

```diff
@@ -0,0 +1,94 @@
+# Custom OpenShift Versions (Dynamic z-stream Support)
+
+As part of [Hive Integration](https://github.com/openshift/assisted-service/blob/master/docs/Kube-API.md), a means to dynamically (without restarting the service) update an OCP version is required.
+This is mainly useful for disconnected environments that use the kube-api support. E.g. updating the release image to a new z-stream version. 
```

Comment:
I would tweak this wording to target environments with mirrored releases. Customers mirror content for more reasons than just having disconnected networks.